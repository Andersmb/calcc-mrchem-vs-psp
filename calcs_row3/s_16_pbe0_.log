Starting job 5688407 on c1-[24,36,38],c2-49,c3-18,c5-49 at Fri May 20 17:53:59 CEST 2022

Copying s_16_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying s_16_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688407      s_16_pbe0_        180                                             00:22:33      0:0 
5688407.bat+      batch         36        1   00:00:02          0   00:00:02   00:22:33      0:0 
5688407.ext+     extern        180        6   00:00:00          0   00:00:00   00:22:33      0:0 
5688407.0      mrchem.x        180       30   00:21:47         29   00:51:03   00:21:55      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688407                                                                          
5688407.bat+     19332K          0     19332K        0              0          0 
5688407.ext+          0          0          0        0              0          0 
5688407.0      5294044K         29 3440566135        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688407                                                                                               
5688407.bat+       19.43M               0         19.43M        0.06M                0          0.06M 
5688407.ext+        0.00M               0          0.00M            0                0              0 
5688407.0           6.77M               1          6.74M       63.09M               16         33.30M 

Job 5688407 completed at Fri May 20 18:16:32 CEST 2022
