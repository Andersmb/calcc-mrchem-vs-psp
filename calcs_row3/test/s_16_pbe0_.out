                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000       -3.339481
    1       S            :        0.000000        0.000000        0.722863
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              10
 Beta electrons          :                                               9
 Total electrons         :                                              19
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999998626e-01
    1    1    a          :                              9.999999999216e-01
    2    1    a          :                              1.000000000628e+00
    3    1    a          :                              1.000000002243e+00
    4    1    a          :                              1.000000000050e+00
    5    1    a          :                              9.999996102587e-01
    6    1    a          :                              1.000000001197e+00
    7    1    a          :                              1.000000000550e+00
    8    1    a          :                              9.999999999399e-01
    9    1    a          :                              1.000000000313e+00
   10    1    b          :                              9.999999992975e-01
   11    1    b          :                              9.999999950887e-01
   12    1    b          :                              1.000000000628e+00
   13    1    b          :                              1.000000000022e+00
   14    1    b          :                              1.000000000050e+00
   15    1    b          :                              9.999999996637e-01
   16    1    b          :                              1.000000000331e+00
   17    1    b          :                              1.000000000550e+00
   18    1    b          :                              9.999999999474e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                402.989263689543
 E-N energy              :            (au)               -987.427623816688
 Coulomb energy          :            (au)                196.684401675582
 Exchange energy         :            (au)                 -6.742485542673
 X-C energy              :            (au)                -20.769305431446
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 11.815838459922
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -415.265749425683
 Nuclear energy          :            (au)                 11.815838459922
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.034499109658e+02
                         :      (kcal/mol)             -2.531686393982e+05
                         :        (kJ/mol)             -1.059257587930e+06
                         :            (eV)             -1.097843087568e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -88.138293476159
    1    1    a          :            (au)                 -7.694570035363
    2    1    a          :            (au)                 -5.655558895017
    3    1    a          :            (au)                 -5.634138152743
    4    1    a          :            (au)                 -5.633563929493
    5    1    a          :            (au)                 -2.073588400673
    6    1    a          :            (au)                 -0.539458821169
    7    1    a          :            (au)                 -0.187583749206
    8    1    a          :            (au)                 -0.153321572327
    9    1    a          :            (au)                 -0.137054731315
   10    1    b          :            (au)                -88.133051963717
   11    1    b          :            (au)                 -7.688994284004
   12    1    b          :            (au)                 -5.636460286925
   13    1    b          :            (au)                 -5.631052877386
   14    1    b          :            (au)                 -5.630707028950
   15    1    b          :            (au)                 -2.073365887880
   16    1    b          :            (au)                 -0.492826991304
   17    1    b          :            (au)                 -0.136800196603
   18    1    b          :            (au)                 -0.121044193852
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -231.391435474086
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.358899e+00        -403.449910965761         -4.034499e+02
    1          3.122986e-01        -405.487615230861         -2.037704e+00
    2          1.369405e-01        -405.555890559568         -6.827533e-02
    3          6.683018e-02        -405.565470567474         -9.580008e-03
    4          2.933154e-02        -405.567590117122         -2.119550e-03
    5          2.962465e-02        -405.568673880074         -1.083763e-03
    6          9.120959e-03        -405.568782757634         -1.088776e-04
    7          1.269702e-02        -405.568886847563         -1.040899e-04
    8          1.026461e-02        -405.568934922660         -4.807510e-05
    9          4.137020e-02        -405.569037939551         -1.030169e-04
   10          3.123925e-02        -405.569173090325         -1.351508e-04
   11          3.526291e-02        -405.569312158513         -1.390682e-04
   12          1.864439e-02        -405.569359112792         -4.695428e-05
   13          8.130556e-03        -405.569377664674         -1.855188e-05
   14          2.887282e-03        -405.569387145933         -9.481259e-06
   15          1.135153e-03        -405.569388827138         -1.681205e-06
   16          5.237347e-04        -405.569389323747         -4.966093e-07
   17          5.687733e-04        -405.569389516763         -1.930156e-07
   18          2.020900e-04        -405.569389580699         -6.393572e-08
   19          1.001743e-04        -405.569389598893         -1.819433e-08
---------------------------------------------------------------------------
                      SCF converged in 19 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000       -3.339481
    1       S            :        0.000000        0.000000        0.722863
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                404.987226022120
 E-N energy              :            (au)               -989.616221055696
 Coulomb energy          :            (au)                194.667341280885
 Exchange energy         :            (au)                 -6.716844324338
 X-C energy              :            (au)                -20.706729981786
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 11.815838459922
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -417.385228058815
 Nuclear energy          :            (au)                 11.815838459922
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.055693895989e+02
                         :      (kcal/mol)             -2.544986323099e+05
                         :        (kJ/mol)             -1.064822278276e+06
                         :            (eV)             -1.103610482487e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -88.960515725972
    1    1    a          :            (au)                 -7.939418639441
    2    1    a          :            (au)                 -5.884787508021
    3    1    a          :            (au)                 -5.868549084772
    4    1    a          :            (au)                 -5.868094218714
    5    1    a          :            (au)                 -2.045131332097
    6    1    a          :            (au)                 -0.636194395530
    7    1    a          :            (au)                 -0.264998205165
    8    1    a          :            (au)                 -0.242082325656
    9    1    a          :            (au)                 -0.218063598576
   10    1    b          :            (au)                -88.955239817760
   11    1    b          :            (au)                 -7.934224854180
   12    1    b          :            (au)                 -5.866213781050
   13    1    b          :            (au)                 -5.865601761883
   14    1    b          :            (au)                 -5.865006908742
   15    1    b          :            (au)                 -2.045309301006
   16    1    b          :            (au)                 -0.586231999054
   17    1    b          :            (au)                 -0.229172328580
   18    1    b          :            (au)                 -0.205192771713
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -235.480028557912
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000018       -0.000002       -4.296343
 Magnitude               :            (au)                        4.296343
                         :         (Debye)                       10.920213
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.547371
 Magnitude               :            (au)                        1.547371
                         :         (Debye)                        3.933025
---------------------------------------------------------------------------
 Total vector            :       -0.000018       -0.000002       -2.748972
 Magnitude               :            (au)                        2.748972
                         :         (Debye)                        6.987188
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 21m 48s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
